src/components/snackbar/snackbar.tsx

```diff
@@ -67,9 +67,9 @@ export class Snackbar {
 
     /**
      * The amount of time in milliseconds to show the snackbar.
-     * If set to `null`, the snackbar will be persistent.
+     * If set to `null` or any value less than or equal to 0, the snackbar will be persistent.
      * This means:
-     * - either the end user will need to close is manually,
+     * - either the end user will need to close it manually,
```

Comment:
@lattaai1 We use atomic commits, so this typo fix should be placed in its own commit.